static/js/pages/profiles/actions/EditProfileBtn.tsx

```diff
@@ -11,25 +11,17 @@ interface Props {
   label?: string;
```

Comment:
We are left with unused props in the interface up here.

Reply:
missed that one, removed now.